### PR TITLE
Try to fix migration

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/dev#6506

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf [hunk 1]
```diff
@@ -49,6 +49,9 @@ fi
 
 if [[ -n $database ]]; then
     # we still use mariadb55, this is wrong we need rh-mariadb105
+
+    # Make sure loglevel is set to 2 (see https://github.com/nextcloud/server/pull/27334)
+    OCC "config:system:set loglevel --value=2 --type=integer"
```
Comment (stephdl) [inline root/etc/e-smith/events/actions/nethserver-nextcloud-conf:54]: This is the default log warning and normally after the upgrade it is written in config.php of nextcloud

https://gist.github.com/stephdl/afd6b203068930caf0cba9e2580a2d62

However it should not beak

File: root/usr/local/sbin/occ [hunk 2]
```diff
@@ -22,4 +22,4 @@
 
 # OCC wrapper running PHP from SCL
 
-runuser -u apache -- scl enable rh-php73 --  php -d memory_limit=512M  /usr/share/nextcloud/occ "$@"
+runuser -u apache -- scl enable rh-php73 --  php -d memory_limit=1024M  /usr/share/nextcloud/occ "$@"
```
Comment (stephdl) [inline root/usr/local/sbin/occ:25]: I did not know, it should help

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf [hunk 3]
```diff
@@ -87,6 +90,11 @@ if [[ -n $database ]]; then
 
     if  ! databaseTest ; then
         OCC config:system:set dbhost --value="localhost:/run/rh-mariadb105-mariadb/nextcloud-mysql.sock" --type="string"
+        if [[ $? -gt 0 ]]; then
```
Comment (stephdl) [inline root/etc/e-smith/events/actions/nethserver-nextcloud-conf:93]: I do not understand here on[ line 12 ](https://github.com/NethServer/nethserver-nextcloud/blob/2393384fb82abf866cb5a8380c318811d6ae902c/root/etc/e-smith/events/actions/nethserver-nextcloud-conf#L12)

We disable maintenance mode and enable accessibility with the function `exitOnError` I believe we do not need it, only exitOnError if we cannot configure the mysql socket

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf (hunk 1 above)
Comment (gsanchietti) [reply root/etc/e-smith/events/actions/nethserver-nextcloud-conf:54]: On a machine coming from an old nextcloud version, the parameter was set to `0` and it broke things :)

File: root/etc/e-smith/events/actions/nethserver-nextcloud-conf (hunk 3 above)
Comment (gsanchietti) [reply root/etc/e-smith/events/actions/nethserver-nextcloud-conf:93]: You're right.
Fixed

**Review by stephdl — APPROVED** [review]

File: root/usr/local/sbin/occ (hunk 2 above)
Comment (gsanchietti) [reply root/usr/local/sbin/occ:25]: Let's hope so :crossed_fingers:

**Pull request merged**